dbbuild/sources/zhu2023/table-s1.csv

```diff
@@ -0,0 +1,502 @@
+﻿Supplementary Table 1. The detailed information of 500 moderately polymorphic microhaplotypes. All MHs Ae values are about 5.,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+ID,bp,Ae,Heterozygosity,Discrimination power,SNP_num,Allele_num,SNP_INFO,Allele_INFO(CHS),,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+CHSCHR01_0012052,331,5,0.80952381,0.924444444,8,10,3098384,3098489,3098571,3098629,3098644,3098660,3098661,3098715,GTGCGCGC:0.004761904761904762,GTACGCAC:0.2523809523809524,GTATGTGC:0.19523809523809524,GTACGCAT:0.24761904761904763,GCACACAC:0.1761904761904762,TTGCGCGC:0.04285714285714286,GTACGCGC:0.06190476190476191,GCACGCAC:0.009523809523809525,GCACGCAT:0.004761904761904762,GTACGTAC:0.004761904761904762,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+CHSCHR10_0290121,340,5,0.866666667,0.923356009,5,8,72046476,72046512,72046609,72046798,72046816,CACAC:0.2,CACCC:0.32857142857142857,CGCCC:0.08571428571428572,TACCC:0.10952380952380952,TACAC:0.10952380952380952,CGCCT:0.14285714285714285,CGCAC:0.009523809523809525,CATAC:0.014285714285714285,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+CHSCHR21_0003240,341,5,0.761904762,0.920272109,8,13,15243139,15243242,15243263,15243339,15243353,15243433,15243447,15243480,CGCAGCCG:0.08095238095238096,CCCCGCCG:0.06190476190476191,CGCCGCCG:0.21904761904761905,CGGAGCTC:0.319047619047619,CGGAATTC:0.19047619047619047,TGGAGCTC:0.03333333333333333,CGGCATTC:0.01904761904761905,CGGCGCTC:0.04285714285714286,CGGAGCCC:0.009523809523809525,CCCAGCCG:0.009523809523809525,TGGCGCTC:0.004761904761904762,CGGAGCCG:0.004761904761904762,CGGAATCG:0.004761904761904762,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,

```

Comment:
Authors did not attempt to associate SNPs with RSIDs, they provided only GRCh37(!) coordinates.